native/src/main/java/io/ballerina/stdlib/constraint/Constraints.java

```diff
@@ -89,4 +89,7 @@ private static AbstractAnnotations getAnnotationImpl(Type type, List<String> fai
             return new TypeAnnotations(failedConstraints);
         }
     }
+
+    private Constraints() {
```

Comment:
Why do we need this constructor?

Reply:
This is a coding practice to have private constructors in classes to avoid instantiating (to hide the implicit public one). 
Not related to the fix.